### ISSUE TITLE
Sequence classification with Lit-LLaMA

### ISSUE THREAD

**Issue by VukVukovic** [Lightning-AI/lit-llama#331]
I'm trying to use Lit-LLaMA with LoRA for a sequence classification problem involving a dataset of tweets, similar to `ought/raft`. A straightforward approach would be to follow the idea described in the T5 paper, which treats the problem as a text generation task and utilizes the logits/probabilities of the class label tokens. My question is whether this approach is suitable for (Lit-)LLaMA as well, or if there is a better alternative for tackling this problem.

**Comment by maadnfritz** [comment]
also interested
